.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this shouldn't be necessary

Reply:
:sweat_smile: :sweat_smile: :sweat_smile: